torchgen/api/lazy.py

```diff
@@ -203,12 +203,16 @@ class LazyArgument:
     # TODO: this is lies, it is false for symint list
     is_symint_or_list: bool
 
+    # Whether or not we are treating this as symint or not
+    symint: bool
+
     # true if this argument is or contains a lazy IR value
     is_lazy_value: bool
 
-    def __init__(self, arg: Argument, properties: "LazyIrProperties"):
+    def __init__(self, arg: Argument, properties: "LazyIrProperties", *, symint: bool):
```

Comment:
why do you add '*' here?

Reply:
There's only one boolean value so technically there is no chance for confusion, but I like forcing booleans to be passed as kwargs to avoid boolean blindness in the event someone decides to add another boolean argument

Reply:
ah, yea that sounds like a good policy.